docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -0,0 +1,78 @@
+# CheckBox
+CheckBox is a common component and describes what action will occur when you select it.  
+A CheckBox can only contain an icon, and can be created using style.
+
+![CheckBox1](./media/CheckBox1.png)
+
+## Create with Style
+
+To create a CheckBox applying style, follow these steps:
+
+1. Create CheckBox using the default constructor:
+
+    ```cs
+    CheckBox utilityCheckBox = new CheckBox();
+    ```
+
+2. Apply style to the RadioButton:
+
+    ```cs
+    ButtonStyle utilityStyle = new ButtonStyle
+    {
+        Icon = new ImageViewStyle
+        {
+            Size =  new Size(48, 48),
+            ResourceUrl = new Selector<string>
+            {
+                Normal = CommonResource.GetFHResourcePath() + "9. Controller/controller_btn_check_off.png",
+                Selected = CommonResource.GetFHResourcePath() + "9. Controller/[Controller] App Primary Color/controller_btn_check_on_24c447.png",
+                Disabled = CommonResource.GetFHResourcePath() + "9. Controller/controller_btn_check_off.png",
+                DisabledSelected = CommonResource.GetFHResourcePath() + "9. Controller/[Controller] App Primary Color/controller_btn_check_on_24c447.png",
```

Comment:
Just for checking. The image names and the paths are unusual. Would you please confirm them?

- `9. Controller/controller_btn_check_off.png`
- `9. Controller/[Controller] App Primary Color/controller_btn_check_on_24c447.png`

How about changing the code to show only the image names, so that it doesn't give any further information about your local work space?

Reply:
Fixed